installation/resources/crds/eventing/eventingbackends.eventing.kyma-project.io.crd.yaml

```diff
@@ -47,10 +47,10 @@ spec:
           metadata:
             type: object
           spec:
-            description: EventingBackendSpec defines the desired state of EventingBackend
+            description: EventingBackendSpec defines the desired state of EventingBackend.
```

Comment:
just wondering: why dot here and other places like line 57 not ?

Reply:
Its just linter magic. The description comes from the comment in the _types.go file. The dot was not added [here](https://github.com/kyma-project/kyma/blob/main/components/eventing-controller/api/v1alpha1/eventingbackend_types.go) and hence not reflected in the CRD